MG-91 - Update Domain Management docs (#99)
absmach/magistrala-docs: docs/user-guide/domain-management/billing.md

> Please note that this service is only available to **Magistrala Cloud** users.

> The **Billing** service allows users to view and subscribe to billing plans, manage payment methods and billing information and download generated invoices and receipts.  

Reviewer: ```suggestion
The **Billing** service allows users to view and subscribe to billing plans, manage payment methods and billing information and download generated invoices.  
```

> Magistrala uses a trusted **third-party payment processor (Paddle)** to handle sensitive payment information.  

Reviewer: ```suggestion
Magistrala uses a trusted **third-party payment processor** to handle sensitive payment information.  
```

The subscription section handles the domain's subscription plan and shows the status of the subscription.

To update a billing plan from the current `Free` plan or any other, click `Update`, which will display a list of available subscription plans.

![Billing-plan-addition](../../img/billing/choose.png)

![Billing-plans](../../img/billing/sub-plans.png)

> The subscription status can be either **active**, where the user has already been charged, or **pending**, where the process is not yet finished and the subscription is inactive.

Reviewer: ```suggestion
The subscription status can be **active**, meaning the user has an active subscription, **waiting for payment**, meaning the payment process has not yet been completed, or **canceled**, meaning the subscription has been terminated.
```
